src/libs/Pusher/pusher.js

```diff
@@ -358,6 +360,46 @@ function reconnect() {
     socket.connect();
 }
 
+/**
+ * Abstraction around subscribing to private user channel events. Handles all logs and errors automatically.
+ *
+ * @param {String} eventName
+ * @param {String} accountID
+ * @param {Function} onEvent
+ * @param {Boolean} isChunked
+ */
+function subscribeToPrivateUserChannelEvent(eventName, accountID, onEvent, isChunked = false) {
+    const pusherChannelName = `private-encrypted-user-accountID-${accountID}${CONFIG.PUSHER.SUFFIX}`;
```

Comment:
NAB: We should consider making this prefix a constant like we do in Web: https://github.com/Expensify/Web-Expensify/blob/25038455beaab04f40fe01f595608655d67e5a96/site/lib/PushNotifications.jsx#L36-L42